pkg/cloud/services/networking/network.go

```diff
@@ -102,23 +102,23 @@ func (s *Service) ReconcileExternalNetwork(openStackCluster *infrav1.OpenStackCl
 	case 0:
 		// Not finding an external network is fine
 		openStackCluster.Status.ExternalNetwork = &infrav1.NetworkStatus{}
-		s.scope.Logger().Info("No external network found - proceeding with internal network only")
+		s.scope.Logger().V(2).Info("No external network found - proceeding with internal network only")
 		return nil
 	case 1:
 		openStackCluster.Status.ExternalNetwork = &infrav1.NetworkStatus{
 			ID:   networkList[0].ID,
 			Name: networkList[0].Name,
 			Tags: networkList[0].Tags,
 		}
-		s.scope.Logger().Info("External network found", "network id", networkList[0].ID)
+		s.scope.Logger().V(2).Info("External network found", "id", networkList[0].ID)
```

Comment:
For the same reason as above I think this is ok at level 2, whereas normally it would be lower.